python/py-schema/Portfile

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-schema
+version             0.7.7
+revision            0
+
+license             MIT
+maintainers         nomaintainer
+
+description         Simple data validation library
+long_description    schema is a library for validating Python data structures, \
+                    such as those obtained from config-files, forms, external services or command-line parsing, \
+                    converted from JSON/YAML (or something else) to Python data-types.
+
+homepage            https://github.com/keleshev/schema
+
+checksums           rmd160  441eb7557fefade868664995c3ddfc66c09e91e0 \
+                    sha256  7da553abd2958a19dc2547c388cde53398b39196175a9be59ea1caf5ab0a1807 \
+                    size    44245
+
+python.versions     38 39 310 311 312
```

Comment:
please only add the subport you're actually using (i.e. 312). Earlier versions can always be added later if there is a need.